ang/crmFieldMetadata/crmRenderCheckbox.html

```diff
@@ -1,7 +1,22 @@
+<!-- TODO: Is this necessary? Under what circumstances would there be no options? -->
 <input ng-if="!field.options.length" type="checkbox" ng-model="$parent.model"  name="{{field.name}}" id="{{prefix}}{{field.name}}" ng-required="field.required" />
 
+<!-- Because we are using multiple HTML elements (the checkboxes) to represent
```

Comment:
We finished this conversation in meatspace and MZD agreed the revised comment was adequate.